### ISSUE TITLE
zmalloc.c:265: error: conflicting types for 'zmalloc_get_rss'

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ashish21590/redis#437]
```
View zmalloc.c:265

In zmalloc.h:42  - size_t
In zmalloc.c:265 - float

While compiling error appear:
zmalloc.c:265: error: conflicting types for 'zmalloc_get_rss'
zmalloc.h:42: error: previous declaration of 'zmalloc_get_rss' was here

```

Original issue reported on code.google.com by `vita...@gmail.com` on 20 Jan 2011 at 2:57

**Comment by GoogleCodeExporter** [comment]
```
Hello. Can't reproduce. What Redis version?
```

Original comment by `anti...@gmail.com` on 20 Jan 2011 at 8:37

**Comment by GoogleCodeExporter** [comment]
```
Oh, sorry, redis 2.2.0rc3
```

Original comment by `vita...@gmail.com` on 20 Jan 2011 at 9:50

**Comment by GoogleCodeExporter** [comment]
```
Error appear while compiling on FreeBSD
```

Original comment by `vita...@gmail.com` on 20 Jan 2011 at 9:53

**Comment by GoogleCodeExporter** [comment]
```
Ok was a BSD specific issue, fixed in 2.2 branch...

The fix will appear on RC4, for now just edit zmalloc.c and change "float" in 
line 265 with "size_t" and everything will work well.

Thanks for reporting,
Cheers,
Salvatore
```

Original comment by `anti...@gmail.com` on 20 Jan 2011 at 10:27
- Changed state: **Fixed**
